docs/guides/instances.mdx

```diff
@@ -0,0 +1,112 @@
+
+<span id="switcher"></span>
+1. To add an instance to your account, first ensure you have the correct account and region selected in the account switcher in the header.
+        * You can have (or be part of) multiple IBM Cloud accounts.  If you don't see the instances or jobs you're expecting, try a different account.
+        * You can access either region by using the same API key, but you can only see and access the instances that were created in the region that you're logged in to.
+        * If you want to create an Open instance, you must choose the us-east region.
+
+        ![The IBM Quantum Platform header is shown.  The account switcher is immediately to the right of the search bar.  The region switcher is immediately to the right of the account switcher.](/docs/images/guides/cloud-setup/IQP-Header.svg 'IBM Quantum Platform header')
+
+   <span id="regions"></span>
+   <Admonition type="note" title="About regions">
+   The instance will be created in the selected region. This region determines these important characteristics:
+     * Which QPUs are available.
+     * Where the jobs' classical computation, such as compilation, takes place.
+     * Where the user's workflow data remains.
+
+    The workflow data includes the input circuits, circuit parameters, and quantum computation results.  It does not include metadata, such as number of shots, size of circuits, or other job statistics.
+
+   To avoid exposure of the user's workflow data to a global Distributed Denial of Service protection layer, you can use [Virtual Private Endpoints](https://cloud.ibm.com/docs/overview?topic=overview-endpoints-support). `qiskit-ibm-runtime` supports this through the [private_endpoint parameter](/docs/api/qiskit-ibm-runtime/qiskit-runtime-service).
+   </Admonition>
+
+1. From the dashboard Instances pane, click **View all**, or go to the [Instances](https://quantum.cloud.ibm.com/instances) page from the main menu.
+1. From the Instances page, click **Create instance**.
+    * Enter a name and optionally add tags.
+    * If you have access to more than one resource group in IBM Cloud&reg;, you can choose to change the resource group this instance belongs to. See [Managing resource groups](https://cloud.ibm.com/docs/account?topic=account-rgs&interface=ui) and [Giving access to resources in resource groups](https://cloud.ibm.com/docs/account?topic=account-rgs_manage_access) for more information.
+1. Select the pricing plan this instance is associated with (for example, Pay-as-you-go). Depending on the pricing plan selected, you can specify other options, such as the cost limit or the number of minutes to allocate to this instance. Note that If this instance is not associated with a free plan, a cost will be incurred when this instance is used to run workloads.
```

Comment:
```suggestion
1. Select the pricing plan this instance is associated with (for example, Pay-As-You-Go). Depending on the pricing plan selected, you can specify other options, such as the cost limit or the number of minutes to allocate to this instance. Note that If this instance is not associated with a free plan, a cost will be incurred when this instance is used to run workloads.
```
It hurts my heart to correct this since you know how I feel about this capitalization